src/auction-salt/auction-salt.ts

```diff
@@ -27,7 +27,7 @@ export class AuctionSalt {
             throw new Error('salt should be less 18 bytes')
         }
 
-        this.salt = auction.salt || randomInt(10000).toString()
+        this.salt = auction.salt || randomIntString(5)
```

Comment:
📓 the salt is 144 bit because

```
uint256 private constant _SALT_MASK              = 0x0000000000000000000000000000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF; // prettier-ignore
```

0x0000000000000000000000000000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF
which in binary

<img width="497" alt="Screenshot 2023-04-11 at 09 20 15" src="https://user-images.githubusercontent.com/30667180/231063336-8dd35960-0ccd-43a7-9b61-b2cea774d7ac.png">

which means the max value is 2^144 -1, which is far far away from out current function